### PR TITLE
feat: accept size config as input ✨

### LINKED ISSUE #11: Pre-defined size of PRs
The current definition of sizes are as 
```
const sizes = {
  0: "XS",
  10: "S",
  30: "M",
  100: "L",
  500: "XL",
  1000: "XXL"
};
```
However it's possible to define `SIZES` in yaml file and then parse them from `process.env.SIZES`. Would be nice to have this feature. Thank you.

### PR DESCRIPTION
**Pull request by mrchief**
Fixes #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,9 +45,49 @@ The following environment variables are supported:
...
+on: pull_request
+jobs:
+  size-label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: size-label
+        uses: "pascalgn/size-label-action@v0.4.2"
         env:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-          IGNORED: ".*\n!.gitignore\nyarn.lock\ngenerated/**"
```
Comment (davidparsson) [inline README.md:50]: Should this have been removed?

Comment (mrchief) [reply README.md:50]: 🤷‍♂️

Comment (davidparsson) [reply README.md:50]: Nevermind, I misread the diff

**Comment by davidparsson** [comment]
@pascalgn, is there anything preventing this from being merged?

**Comment by pascalgn** [comment]
> @pascalgn, is there anything preventing this from being merged?

Yes, time 😉   Thanks for the reminder! And thanks to @mrchief for the PR, it's merged now! 👍

**Comment by davidparsson** [comment]
I'm well aware.. Thanks!

**Pull request merged**